Sample app is not working.

Hello.
I was trying to use your library in small test application and failed.
What I was doing:
1. Created simple ConsoleApplication1.
2. Added new ClassLibrary1
3. Installed Nuget package CodeGeneration.Roslyn 0.1.57 to class library.
4. Added DuplicateWithSuffixAttribute and DuplicateWithSuffixGenerator classes from README.md to my class libraty.
(4.1.) Fixed a little DuplicateWithSuffixGenerator (IReadonlyCollection changed to SyntaxList).
5. Added Foo class to my ConsoleApplication1.
6. Added     [DuplicateWithSuffix("A")] attribute to Foo class.
7. Set Custom Tool property of Foo.cs file to `MsBuild:GenerateCodeFromAttributes`.
7. Added `var fooA = new FooA();` line to Main method.
8. Tried to build my test solution.

And unfortunately, ConsoleApplication1 is not building with error `error CS0246: The type or namespace name 'FooA' could not be found (are you missing a using directive or an assembly reference?)`. No other errors or warnings.
My guess is that GenerateCodeFromAttributes task is not launching. 
If I can provide any additional information, please let me know.

Thanks.

--- a/README.md
+++ b/README.md
@@ -50,16 +50,16 @@ public class DuplicateWithSuffixGenerator : ICodeGenerator
         this.suffix = (string)attributeData.ConstructorArguments[0].Value;
     }
 
-    public Task<IReadOnlyList<MemberDeclarationSyntax>> GenerateAsync(MemberDeclarationSyntax applyTo, Document document, IProgress<Diagnostic> progress, CancellationToken cancellationToken)
+    public Task<SyntaxList<MemberDeclarationSyntax>> GenerateAsync(MemberDeclarationSyntax applyTo, Document document, IProgress<Diagnostic> progress, CancellationToken cancellationToken)
     {
-        var results = new List<MemberDeclarationSyntax>();
+        var results = SyntaxFactory.List<MemberDeclarationSyntax>();
 
         var applyToClass = (ClassDeclarationSyntax)applyTo;
         var copy = applyToClass
             .WithIdentifier(SyntaxFactory.Identifier(applyToClass.Identifier.ValueText + this.suffix));
-        results.Add(copy);
+        results = results.Add(copy);
 
-        return Task.FromResult<IReadOnlyList<MemberDeclarationSyntax>>(results);
+        return Task.FromResult<SyntaxList<MemberDeclarationSyntax>>(results);
     }
 }
 ```

--- a/README.md
+++ b/README.md
@@ -99,6 +99,11 @@ Any file that uses this attribute should have its Custom Tool property set to
 `MSBuild:GenerateCodeFromAttributes` so that when you save the file or compile
 the project, the code generator runs and acts on your attribute.
 
+Install the [CodeGeneration.Roslyn.BuildTime][BuildTimeNuPkg] package into the
+project using your attribute:
+
+    Install-Package CodeGeneration.Roslyn.BuildTime -Pre
+
 You can then consume the generated code at design-time:
 
 ```csharp
@@ -159,3 +164,4 @@ dependencies it needs *besides those delivered by the `CodeGeneration.Roslyn.Bui
 This will typically mean it has your attributes assembly and your generator assembly.
 
 [NuPkg]: https://nuget.org/packages/CodeGeneration.Roslyn
+[BuildTimeNuPkg]: https://nuget.org/packages/CodeGeneration.Roslyn.BuildTime

--- a/README.md
+++ b/README.md
@@ -24,6 +24,16 @@ In the following example, we define a pair of classes which exactly duplicates
 any class the attribute is applied to, but adds some suffix to the name on the copy.
 
 ```csharp
+using System;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using CodeGeneration.Roslyn;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Validation;
+
 [AttributeUsage(AttributeTargets.Class, Inherited = false, AllowMultiple = true)]
 [CodeGenerationAttribute(typeof(DuplicateWithSuffixGenerator))]
 [Conditional("CodeGeneration")]

